packages/opaca/opaca.0.1.3/opam

```diff
@@ -0,0 +1,15 @@
+opam-version: "1.2"
+name: "opaca"
+authors: "Mia Kathage"
+maintainer: "Mia Kathage <siboru@googlemail.com>"
+homepage: "https://github.com/spreadLink/opaca"
+bug-reports: "https://github.com/spreadLink/opaca/issues"
+dev-repo: "https://github.com/spreadLink/opaca.git"
+doc: "https://spreadlink.github.io/opaca/"
+license: "MIT License"
+build: ["jbuilder" "build"]
+depends: [
+  "ocamlfind" {build}
```

Comment:
a minor nit that's not a blocker; you probably don't need ocamlfind for this, just jbuilder.  (jbuilder will pull in ocamlfind if it needs it)

Reply:
Thanks :)
Tho I need to learn to read docs properly it seems 😅